auto_backup/models/db_backup.py

```diff
@@ -240,10 +240,10 @@ def cleanup(self):
                 oldest = self.filename(now - timedelta(days=rec.days_to_keep))
```

Comment:
I think the new philosophy messes with this approach. What do you think ?

Reply:
I kept the remote part as it is because I don't use it myself, but I think you're right, it doesn't make sense to do A remotely and B locally. Will add this to my TODO list